combat-trainer.lic

```diff
@@ -1405,7 +1405,7 @@ class AbilityProcess
     @barb_buffs.each do |name|
       ability_data = get_data('spells').barb_abilities[name]
       Flags.add("ap-#{name}-expired", ability_data['expired_message'])
-      Flags["ap-#{name}-expired"] = true
+      Flags["ap-#{name}-expired"] = true unless DRSpells.active_spells[name]
```

Comment:
What will this do if you don't have `med power` or Powermonger?

Reply:
Fuck Barbarian skills.

Reply:
Yeah, tested this, you need at least Power to see spells being active. What do you think could be done to handle this then?

Reply:
PM won't do anything for this, the person just needs POWER. Actually, I don't think this is a big deal, it just won't see the spell as active and it will use it on startup. People that can use Meditations in combat 100% have Power at least, and it will function the same as it did before. The skill will attempt to be used and since it's already active, it will just reset the flag. So, actually this isn't an issue.

Reply:
I agree with all of the above